src/pod.ts

```diff
@@ -437,6 +440,13 @@ export interface AbstractPodProps extends base.ResourceProps {
    * @default false
    */
   readonly hostNetwork?: boolean;
+
+  /**
+   * Grace period until the pod is terminated
+   *
+   * @default Duration.seconds(30)
+   */
+  readonly terminationGracePeriodSeconds?: Duration;
```

Comment:
It doesn't have to be seconds actually, `Duration` has a other APIs :)

```suggestion
  readonly terminationGracePeriod?: Duration;
```

Reply:
According to k8s api, it's in seconds: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-termination:~:text=terminationGracePeriodSeconds
Unless I misunderstood :)

Reply:
Yes, eventually you'll have to convert it to seconds (like you already are). But a user of cdk8s-plus can pass any `Duration` object. For example:

```ts
terminationGracePeriodSeconds: Duration.hours(1)`
```

This is great, cdk8s-plus offers a better API then k8s because it does the conversion for you.
So a better name to the property would be just `terminationGracePeriod`.

Reply:
@iliapolo - I changed it to terminationGracePeriod and updated the tests accordingly